client/src/pages/organizations/Approvals.js

```diff
@@ -23,7 +23,9 @@ export default class OrganizationApprovals extends Component {
 						<tbody>
 							{step.approvers.map((position, approverIdx) =>
 								<tr key={position.uuid} id={`step_${idx}_approver_${approverIdx}`} >
-									<td>{position.person && <LinkTo person={position.person} />}</td>
+									<td>{(position.person && position.person.uuid)
```

Comment:
You are nesting `<td>`s here; probably remove the outer `<td>` on this line and the closing `</td>`s at the end of the next two lines?

Reply:
I removed the outermost <td>